tests/plugins/widget/manual/undo.md

```diff
@@ -0,0 +1,13 @@
+@bender-tags: 4.12.2, bug, widget, 3198
+@bender-ui: collapsed
+@bender-ckeditor-plugins: image2, wysiwygarea, toolbar, sourcearea, undo
+
+1. Click on a widget.
+1. Click outside of the editor.
+1. Click on the widget again.
```

Comment:
Can we have more scenarios here? Like focusing text after blurring and clicking undo button after blurring (both triggers the additional snapshot)?

Reply:
>Like focusing text after blurring and clicking undo button after blurring (both triggers the additional snapshot)?

If we don't make any changes to editor content with the fix then undo will always be disabled, so we can't click it after blurring. I'll add the case from main review comment.